docs/presentations/2016_workflows.html

```diff
@@ -14,18 +16,26 @@
 class: inverse, middle
 
 ---
-class: special
-# Planemo: A Scientific Workflow SDK
+class: title
+# Planemo
+## ~~A Galaxy Tool SDK~~
+## A Scientific Workflow SDK
 John Chilton, Aysam Guerler, and the Galaxy Team
 
+???
+
+Oh look, I get to follow Björn again. That guy ruins the curve - when
```

Comment:
I had to google for a bit and still do not understand the phrase 'ruin the curve' correctly I am afraid. Non-native speakers might have problems catching this.